content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,264 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for isolated private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation or jump box. You can use the installer on any system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint.
+
+
+# Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have testing the gateway installation using Ubuntu 20.04.
+
+
+- One IP address for a Kubernetes control plane.
+
+
+- One IP address for a single-node gateway or three IP addresses for a three-node gateway.
+
+
+- Sufficient IPs for application workload services, such as Load Balancer services.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses 192.168.0.0/16. Ensure that the Pod CIDR range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Minimum capacity requirements as follows:
+
+    <br />
+
+    - One node (no HA): 4 CPU, 8 GiB Memory, 60 GiB Storage.
+   
+    - Three nodes (HA): 6 CPU, 12 GiB Memory, 90 GiB Storage.<br /><br />
```

Comment:
@antongisli This might come from the info in your notes in the screenshot below. However, I can't seem to locate the original information for storage or 3 nodes.  

![CleanShot 2023-02-28 at 17 33 39](https://user-images.githubusercontent.com/117382432/222022239-baa98529-b78d-4b5b-994e-9ec3f81080a4.png)

Reply:
@ritawatson the installer CLI enforces the following minimums per node, regardless of whether there is one node or three:

- 4 CPUs
- 4096 MiB memory
- 60 GB storage

Reply:
ok thx Tyler, so something like:
Each node in the PCG cluster requires a machine from MAAS in a ready state, and with the following resources:

- 4 CPUs
- 4096 MiB memory
- 60 GiB storage

that said, @TylerGillson just double checking but did you confirm it works with 4096 MiB? when I was using the old installer, I had issues with <8192 MiB and IIRC @sadysnaat told me that the minimum needed was 8 GiB per node.

Reply:
@antongisli I have not validated that a single node PCG works with those minimums. Will do so today and confirm. That's just what's been baked into the source code since forever.